packages/api/lib/writeRecords/write-granules.js

```diff
@@ -176,6 +177,103 @@ const _writePostgresGranuleViaTransaction = async ({
   return granuleCumulusId;
 };
 
+/**
+ * Update granule status
+ *
+ * @param {Object} params
+ * @param {number} params.granuleCumulusId - Cumulus ID of a granule
+ * @param {string} params.granuleId - granule ID from the payload
+ * @param {Knex} params.knex - knex Client
+ * @param {string} params.status - new status for granule
+ * @returns {Promise}
+ * @throws {Error}
+ */
+async function updateGranuleStatus({
+  granule,
+  knex,
+  status,
+  error = '',
+  errorCause = '',
```

Comment:
```suggestion
  error,
  errorCause,
```

Empty strings and undefined are both falsy, so I think these initializations are unnecessary